Integration test reserve-ports-with-urls fails on windows

the IT ` reserve-ports-with-urls\pom.xml` fails on windows machines with:

```
[ERROR] Failed to execute goal org.codehaus.mojo:build-helper-maven-plugin:3.1.0-SNAPSHOT:reserve-network-port (reserve-ports) on project build-helper-maven-plugin-reverse-port-it: Port names could not be loaded from "URL file://D:/Develop/OpenSource/mojohaus/build-helper-maven-plugin/target/it/reserve-ports-with-urls/names2.txt" -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.codehaus.mojo:build-helper-maven-plugin:3.1.0-SNAPSHOT:reserve-network-port (reserve-ports) on project build-helper-maven-plugin-reverse-port-it: Port names could not be loaded from "URL file://D:/Develop/OpenSource/mojohaus/build-helper-maven-plugin/target/it/reserve-ports-with-urls/names2.txt"
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.MojoExecutionException: Port names could not be loaded from "URL file://D:/Develop/OpenSource/mojohaus/build-helper-maven-plugin/target/it/reserve-ports-with-urls/names2.txt"
    at org.codehaus.mojo.buildhelper.ReserveListenerPortMojo.load (ReserveListenerPortMojo.java:395)
    at org.codehaus.mojo.buildhelper.ReserveListenerPortMojo.loadUrls (ReserveListenerPortMojo.java:382)
    at org.codehaus.mojo.buildhelper.ReserveListenerPortMojo.execute (ReserveListenerPortMojo.java:143)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
```

--- a/src/it/reserve-ports-with-urls/pom.xml
+++ b/src/it/reserve-ports-with-urls/pom.xml
@@ -22,7 +22,7 @@
             </goals>
             <configuration>
               <urls>
-                <url>file://${project.basedir}/names2.txt</url>
+                <url>file:///${project.basedir}/names2.txt</url>
               </urls>
             </configuration>
           </execution>
@@ -35,8 +35,8 @@
             <configuration>
               <outputFile>${project.build.directory}/ports.properties</outputFile>
               <urls>
-                <url>file://${project.basedir}/names1.txt</url>
-                <url>file://${project.basedir}/names2.txt</url>
+                <url>file:///${project.basedir}/names1.txt</url>
+                <url>file:///${project.basedir}/names2.txt</url>
               </urls>
             </configuration>
           </execution>

--- a/src/site/apt/usage.apt.vm
+++ b/src/site/apt/usage.apt.vm
@@ -494,7 +494,7 @@ releasedVersion.qualifier
             <phase>process-resources</phase>
             <configuration>
               <urls>
-                <url>file://${project.basedir}/names1.txt</url>
+                <url>file:///${project.basedir}/names1.txt</url>
                 <url>classpath:port-names.txt</url>
               </urls>
             </configuration>

